### DISCUSSION TITLE
Abstract Controller's addFlash function inquiry

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4804] [Plugins]
$this->addFlash('success', 'Article bien enregistré');    

When I call this function, it pops up at the top of the middle, disappears after 3 seconds, but it doesn't disappear, and I want to keep getting flash messages.
How do I modify duration and location or size?

**Comment by kevinpapst** [comment]
You do not change the nature of a flash message, it is a short living feedback for the user. Nothing else.

Check the functions in `templates/macros/widgets.html.twig`, this might be your thing:
https://github.com/kimai/kimai/blob/main/templates/macros/widgets.html.twig#L253
